sitemap-remoto.sh

```
@@ -0,0 +1,29 @@
+#!/bin/bash
+DOMAIN="https://www.grupomusicalcelula.com"
+OUTPUT="sitemap-remoto.xml"
+
+echo "Rastreando $DOMAIN..."
+
+# Descarga el sitio (solo estructura) y extrae URLs únicas
+wget --spider --recursive --no-verbose --output-file=wget.log "$DOMAIN"
+grep -oP "$DOMAIN[^ ]*" wget.log | sort | uniq > urls.txt
```

Shellcheck (0.11.0)

[error] Use braces when expanding arrays, e.g. ${array[idx]} (or ${var}[.. to quiet).

(SC1087)